akka-distributed-data/src/multi-jvm/scala/akka/cluster/ddata/ReplicatorMapDeltaSpec.scala

```diff
@@ -97,24 +129,44 @@ object ReplicatorMapDeltaSpec extends MultiNodeConfig {
         availableForRemove.toVector(rnd.nextInt(availableForRemove.size))
     }
 
-    (0 to (30 + rnd.nextInt(10))).map { _ ⇒
-      rnd.nextInt(4) match {
+    (0 to (120 + rnd.nextInt(10))).map { _ ⇒
```

Comment:
the trade-off is that with a very small number removals are generated very infrequently, which was the cause of not getting failing case in repeat with previous version of `mergeDelta`